package.json

```diff
@@ -5,7 +5,7 @@
     "ng": "ng",
     "start": "npm run build:misc && ng serve",
     "start:local": "npm run build:misc && ng serve --configuration=local",
-    "build": "ng build",
+    "build": "node --max_old_space_size=4096 node_modules/@angular/cli/bin/ng build",
```

Comment:
Hey Omar let me know what you think about this angular suggested hack, makes the build faster i've noticed

Reply:
can i ask what problem this is trying to address? a build output size issue? is an issue with angular itself?

Reply:
also would this also be required in the `github > deploy.yml` file?